cloudify/tests/test_context.py

```diff
@@ -87,58 +77,77 @@ def redirect_log_to_stdout(logger):
         stdout_log_handler.setLevel(logging.DEBUG)
         logger.handlers = [stdout_log_handler]
 
-    @mock.patch('cloudify.manager.get_rest_client', return_value=MagicMock())
+    @mock.patch('cloudify.manager.get_rest_client')
     def test_get_resource(self, _):
-        resource = self.context.get_resource(
-            resource_path='for_test_bp_resource.txt')
+        with patch('requests.get',
+                   return_value=Mock(ok=True, content=b'Hello from test')):
+            resource = self.context.get_resource(
+                resource_path='for_test_bp_resource.txt')
         self.assertEquals(resource, b'Hello from test')
 
-    def test_get_deployment_resource_priority_over_blueprint_resource(self):
```

Comment:
this tests the ordering of tried urls, which is already tested by `test_download_resource_tried_urls`